static/app/components/replays/player/scrubber.tsx

```diff
@@ -122,10 +97,13 @@ const RangeWrapper = styled('div')`
 `;
 
 const Range = styled(RangeSlider)`
+  & * {
+    height: 100% !important;
+  }
```

Comment:
This is because there's an inner `div`, and then the `input` inside the code `<Slider>` component. we need to override the margin, and make sure the height of everything is 100% tall so the clickable area is bigger than the visible area.